src/main/scala/rocket/PMP.scala

```diff
@@ -23,15 +24,16 @@ object PMP {
 
   def apply(reg: PMPReg): PMP = {
     val pmp = Wire(new PMP()(reg.p))
-    pmp := reg
+    pmp.cfg := reg.cfg
```

Comment:
Seems like I can't do this because chisel3 doesn't like it if names don't match exactly. Happy to take any suggestion other way to do this other than looking at the bundles and seeing what I need to attach one-by-one...

Reply:
I think maybe you can use @terpstra's FixChisel3 operators, but for cases like this with 2-or-so fields, making it explicit seems OK.

Reply:
And I would prefer to get a discussion going about migrating those operators into Chisel 3 before mass proliferation in Rocket Chip.

Reply:
@seldridge I would also like to avoid the proliferation of custom-but-actually-generally-useful operators.

Reply:
I opened a Chisel issue:https://github.com/freechipsproject/chisel3/issues/1316